tests/integration/trigger-file-select-test.js

```diff
@@ -0,0 +1,57 @@
+import { moduleForComponent, test } from 'ember-qunit';
+import hbs from 'htmlbars-inline-precompile';
+import { selectFiles } from 'ember-native-dom-helpers';
+
+moduleForComponent('selectFiles', 'Integration | Test Helper | selectFiles', {
+  integration: true
+});
+
+test('It provids a file under the events target', function(assert) {
+  assert.expect(3);
+  this.onChange = (e) => {
+    assert.equal(e.target.files.length, 1, 'file is present');
+    assert.equal(e.target.files[0].type, 'plain/text', 'file has the set type');
+    assert.equal(e.target.files[0].size, 7, 'file has the set size');
+  };
+
+  this.render(hbs`<input class="target-element" type="file" onchange={{onChange}} />`);
+  selectFiles('.target-element', new Blob(['texters'], { type: 'plain/text' }));
+});
+
+test('Ability to handle multiple files', function(assert) {
+  assert.expect(5);
+  this.onChange = (e) => {
```

Comment:
Should the helper raise if you attempt to add several files to a file input that is not `multiple`?